### PR TITLE
CH-82

### PR DESCRIPTION
**Pull request by michalbella**

### CODE REVIEW

File: src/components/matchesList/MatchesListItem.tsx [hunk 1]
```diff
@@ -55,6 +58,14 @@ const MatchListItem: FC<IMatchListItem> = ({ match, keyValue, filter, loading =
 				{!loading && match.isOpen && isExpanded && (
 					<MatchListContent match={match as any} setVisibleParlayValidationModal={setVisibleParlayValidationModal} />
 				)}
+				<Button
+					style={{ marginTop: '24px' }}
+					type={'primary'}
```
Comment (daniel-hlavaty) [inline src/components/matchesList/MatchesListItem.tsx:63]: You do not need to define type if its 'primary', component has it as default

File: src/components/matchesList/MatchesListStyles.ts [hunk 2]
```diff
@@ -413,6 +414,9 @@ export const NoWrapCenterRow = styled(Row)`
 	display: flex;
 	align-items: center;
 `
+
+export const TeamNames = styled(Row)``
```
Comment (daniel-hlavaty) [inline src/components/matchesList/MatchesListStyles.ts:418]: This can be removed

File: src/components/oddButton/OddButton.tsx [hunk 3]
```diff
@@ -23,22 +23,29 @@ type Props = {
...
 	isMobilePanel?: boolean
 	isHeader?: boolean
+	disabled?: boolean
 }
 
 const OddButton = (props: Props) => {
 	const { t } = useTranslation()
 	const dispatch = useDispatch()
-	const { betOption, match, oddName, setVisibleParlayValidationModal, isMobilePanel, isHeader } = props
+	const { betOption, match, oddName, setVisibleParlayValidationModal, isMobilePanel, isHeader, disabled } = props
```
Comment (daniel-hlavaty) [inline src/components/oddButton/OddButton.tsx:32]: Is there any OddButton that uses this disabled prop?

File: src/components/switchButton/SwitchButton.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+type Props = {
+	option1: string
+	option2: string
+	onClick: (segment: SWITCH_BUTTON_OPTIONS) => void
+}
+
+const SwitchButton = ({ option1, option2, onClick }: Props) => {
+	const [activeSegment, setActiveSegment] = useState(SWITCH_BUTTON_OPTIONS.OPTION_1)
+
```
Comment (daniel-hlavaty) [inline src/components/switchButton/SwitchButton.tsx:13]: Shouldnt this be radiobutton? That we already have, with another style? We might add option 3 4 later and its a bit confusing maybe. You can keep this in, just had to say something :D

File: src/components/ticketList/TicketItem.tsx [hunk 5]
```diff
@@ -98,7 +98,7 @@ const TicketItem = ({ match, oddsInfo }: Props) => {
 					<Icon style={{ marginRight: 0, color: 'white' }} className={`icon icon--${SPORTS_MAP[Number(match?.market.tags?.[0])]?.toLowerCase()}`} />
 				</SC.SportLogo>
 				<SC.TicketStatus matchStatus={getParlayItemStatus(match, isPlayedNow(), t).status}>
-					{getParlayItemStatus(match, isPlayedNow(), t).text}
+					<span>{getParlayItemStatus(match, isPlayedNow(), t).text}</span>
```
Comment (daniel-hlavaty) [inline src/components/ticketList/TicketItem.tsx:101]: did this fix something?

File: src/content/matchesContent/MatchDetailContentStyles.ts [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+					transform: translate(-50%, -50%);
+					width: 8px;
+					height: 8px;
+					border-radius: 50%;
+					background: ${({ theme }) => theme['color-base-state-warning-fg']};
+					animation: ${flicker} 1s infinite;
+				}
+			}
+		`}
+	${(p) =>
```
Comment (daniel-hlavaty) [inline src/content/matchesContent/MatchDetailContentStyles.ts:75]: I see that styles for other matches then ongoing are here, but show match details is visible only in ongoing matches. Is that on purpose?

File: src/content/matchesContent/MatchDetailContent.tsx [hunk 7]
```diff
@@ -1,10 +1,139 @@
...
+	const { chain } = useNetwork()
+	const router = useRouter()
+	const [tab, setTab] = useState(SWITCH_BUTTON_OPTIONS.OPTION_1)
+	const [matchDetailData, setMatchDetailData] = useState<any>(null)
+	const onChangeSwitch = (option: SWITCH_BUTTON_OPTIONS) => {
+		setTab(option)
+	}
+
+	const [fetchMatchDetail] = useLazyQuery(GET_MATCH_DETAIL)
+	const sgpFeesRaw = useSGPFeesQuery(chain?.id as any, {
```
[the remark this answers is not in the document]
Comment (michalbella) [reply src/content/matchesContent/MatchDetailContent.tsx:35]: @jakub-cvacho @daniel-hlavaty for this cases where we use chain?.id we need to retype to as Network
```
export enum Network {
	OptimismMainnet = 10,
	OptimismGoerli = 420,
	ArbitrumOne = 42161
}
```

File: src/components/oddButton/OddButton.tsx (hunk 3 above)
Comment (michalbella) [reply src/components/oddButton/OddButton.tsx:32]: @daniel-hlavaty yes we will use it in cases when ticket is submitting we need to disabled odd buttons. This bug is reported in: CH-451

File: src/components/switchButton/SwitchButton.tsx (hunk 4 above)
Comment (michalbella) [reply src/components/switchButton/SwitchButton.tsx:13]: I think is good name becase you switching between two options and radio buttons is more like switching between 2 and more options. This is more like TabSwitchButton. We dont add more options this is styled only for two options.

File: src/components/ticketList/TicketItem.tsx (hunk 5 above)
Comment (michalbella) [reply src/components/ticketList/TicketItem.tsx:101]: Yes it need to be in span tag becaase there is styles connected to child span tag which generate flickering dot infront of text: playing now.

**Pull request merged**